static/js/src/navigation.js

```diff
@@ -160,6 +160,25 @@ function addGAImpressionEvents(target, category) {
   }
 }
 
+addGADownloadImpressionEvents(".js-download-option", "download-option");
+
+function addGADownloadImpressionEvents(target, category) {
+  var t = [].slice.call(document.querySelectorAll(target));
+  if (t) {
+    t.forEach(function (section) {
+      if (!section.classList.contains("u-hide")) {
+        dataLayer.push({
+          event: "NonInteractiveGAEvent",
+          eventCategory: "www.ubuntu.com-impression-" + category,
+          eventAction: "Display option",
+          eventLabel: section.innerText,
```

Comment:
Do you think that is a little lossly connected? This content could change regularly but might be what your interested in tracking.

Reply:
The goal being to track behavior, the main H1 of the page is a very relevant dimension (especially when it comes to evaluating potential option changes)